lib/screens/account_choice_screen.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import '../utils/app_initialization.dart';
+import '../providers/key_provider.dart';
+import 'lockbox_list_screen.dart';
+import 'login_screen.dart';
+import 'account_created_screen.dart';
+
+/// Screen showing three account setup options:
+/// - Create Account: Generate new Nostr key
+/// - Login: Import existing key
+/// - Continue Without Account: Silent key generation
+class AccountChoiceScreen extends ConsumerWidget {
+  const AccountChoiceScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Account Setup'),
+      ),
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              // Create Account Card
+              _AccountChoiceCard(
+                icon: Icons.person_add,
+                title: 'Create Account',
+                description: 'Generate a new Nostr identity and back it up',
+                onTap: () async {
+                  // Generate new key
+                  final loginService = ref.read(loginServiceProvider);
+                  final keyPair = await loginService.generateAndStoreNostrKey();
+
+                  // Navigate to AccountCreatedScreen with nsec format
+                  if (context.mounted && keyPair.privateKeyBech32 != null) {
+                    Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                        builder: (context) => AccountCreatedScreen(
+                          nsec: keyPair.privateKeyBech32!,
+                          isImported: false,
+                        ),
+                      ),
+                    );
+                  }
+                },
+              ),
+              const SizedBox(height: 16),
+
+              // Login Card
+              _AccountChoiceCard(
+                icon: Icons.login,
+                title: 'Login',
+                description: 'Import your existing Nostr key',
+                onTap: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => const LoginScreen(),
+                    ),
+                  );
+                },
+              ),
+              const SizedBox(height: 16),
+
+              // Continue Without Account Card
+              _AccountChoiceCard(
+                icon: Icons.arrow_forward,
+                title: 'Continue Without Account',
+                description: 'Use the app without a Nostr identity',
```

Comment:
Now that I see this button I think this is misleading. Let's change it to a "Skip For Now" button. Also I don't see an alert with the warning I mentioned in my original prompt about changing devices. Put that in an alert.